code/__DEFINES/~skyrat_defines/construction.dm

```diff
@@ -0,0 +1,3 @@
+		// SKYRAT ADDITION BEGIN - Teshari food define
```

Comment:
its already in a skyrat_define folder, you don't need these.

Reply:
what do you mean?

Reply:
Mhm, but now that it's in a skyrat folder you don't need the comments. Cus you only need them on core files.

Reply:
ahhh right right right